storage/ctab/src/test/java/org/openscience/cdk/io/SDFWriterTest.java

```diff
@@ -370,4 +370,33 @@ public void setProgramName() {
             assertThat(mol, CoreMatchers.containsString("Bioclip"));
         }
     }
+
+    @Test
+    public void optionallyTruncateLongProperties() {
+        StringWriter sw = new StringWriter();
+        try (SDFWriter sdfw = new SDFWriter(sw)) {
+            sdfw.getSetting(MDLV2000Writer.OptWriteDefaultProperties)
+                .setSetting("false");
+            sdfw.getSetting(SDFWriter.OptTruncateLongData)
```

Comment:
Can you also add the mirror test, with the expected outcome when it is set to false? Also relates to my below question.